### PR TITLE
A18-calendar

### PR DESCRIPTION
**Pull request by ghost**
creaton of Calendar

### CODE REVIEW

File: src/jm/html-css/index.html [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+          <div>3 pm</div>
+          <div>4 pm</div>
+        </div>
+        <div class="calendar-container">
+          <div class="calendar-body">
+            <div class="calendar-cell">
+              <div class="card">
+                <img
+                  src="https://tse3.mm.bing.net/th?id=OIP.j_eWpTRqrOOdRgcSmcQv0gHaHp&pid=Api&P=0&h=180"
+                  alt=""
```
Comment (Iregui44) [inline src/jm/html-css/index.html:135]: Don leave alt without text.

File: src/jm/html-css/index.html [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+      </div>
+      <div class="date">
+        <div>Abril</div>
+        <div class="num">20</div>
+        <div class="day-name">TUE</div>
+      </div>
+      <div class="date">
+        <div>Abril</div>
+        <div class="num">21</div>
+        <div class="day-name">WEB</div>
```
Comment (Iregui44) [inline src/jm/html-css/index.html:91]:
```suggestion
        <div class="day-name">WED</div>
```

File: src/jm/html-css/style.css [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+
+.search {
+  margin-bottom: 10px;
+}
+
+.search-input {
+  width: 100%;
+  padding: 10px;
+  border-radius: 50px;
+  box-sizing: border-box;
```
Comment (Iregui44) [inline src/jm/html-css/style.css:43]: I would add a border attribute for make it less noticeable

File: src/jm/html-css/style.css [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+}
+
+.hours {
+  line-height: 131px;
+  font-size: 20px;
+}
+
+.hours > div {
+  border: 1px solid #ddd;
+  padding-left: 250px;
```
Comment (Iregui44) [inline src/jm/html-css/style.css:159]: Align leter to center and a little more gray would be nice.

File: src/jm/html-css/style.css [hunk 5]
```diff
@@ -0,0 +1,308 @@
...
+  align-items: center;
+  justify-content: space-between;
+  background: #f3f3f3;
+  padding: 10px;
+  font-weight: bold;
+}
+
+.calendar-body {
+  display: grid;
+  grid-template-columns: repeat(7, 1fr);
```
Comment (Iregui44) [inline src/jm/html-css/style.css:180]: Add alittle bit space between this components header and body

**Comment by Iregui44** [comment]
You did the sidebar but the instructions said you have to not do it

File: src/jm/html-css/index.html (hunk 1 above)
Comment (Juanmediina) [reply src/jm/html-css/index.html]: added text

File: src/jm/html-css/index.html (hunk 2 above)
Comment (Juanmediina) [reply src/jm/html-css/index.html]: ok

File: src/jm/html-css/style.css (hunk 3 above)
Comment (Juanmediina) [reply src/jm/html-css/style.css:43]: added border

File: src/jm/html-css/style.css (hunk 4 above)
Comment (Juanmediina) [reply src/jm/html-css/style.css:162]: correction made

File: src/jm/html-css/style.css (hunk 5 above)
Comment (Juanmediina) [reply src/jm/html-css/style.css:184]: correction  made

**Review by Iregui44 — APPROVED** [review]

**Comment by Iregui44** [comment]
You have conflicting errors please fix them to be able to marge

**Pull request closed without merging**